config/crd/bases/atlas.mongodb.com_atlasdatabaseusers.yaml

```diff
@@ -91,7 +91,6 @@ spec:
                     type: string
                 required:
                 - name
-                - namespace
```

Comment:
same here, curious how we've modified their existing models

Reply:
Same as above